core/variant/variant_call.cpp

```diff
@@ -1928,18 +1930,28 @@ static void _register_variant_builtin_methods() {
 	bind_method(Basis, inverse, sarray(), varray());
 	bind_method(Basis, transposed, sarray(), varray());
 	bind_method(Basis, orthonormalized, sarray(), varray());
+	bind_method(Basis, orthogonalized, sarray(), varray());
 	bind_method(Basis, determinant, sarray(), varray());
 	bind_methodv(Basis, rotated, static_cast<Basis (Basis::*)(const Vector3 &, real_t) const>(&Basis::rotated), sarray("axis", "angle"), varray());
 	bind_method(Basis, scaled, sarray("scale"), varray());
+	bind_method(Basis, scaled_local, sarray("scale"), varray());
+	bind_method(Basis, scaled_orthogonal, sarray("scale"), varray());
```

Comment:
Does this method also make sense in `Transform2D` that also has `scaled` and `scaled_local` methods?

Reply:
Added `scaled_orthogonal()` to `Transform2D`.